client/options.go

```diff
@@ -267,9 +273,9 @@ func WithLoadBalanceP2C() ReferenceOption {
 	}
 }
 
-func WithLoadBalanceXDSRingHash() ReferenceOption {
+func WithLoadBalance(lb string) ReferenceOption {
```

Comment:
Not only do we need to enumerate, but we also need to leave a generic configuration API?

Reply:
Yes, I realize this is necessary when developing samples. This becomes especially useful when considering Dubbo's high extensibility, we need to make sure our API can support user self-extended LoadBalance policies.